packages/shorebird_cli/lib/src/commands/release/release_command.dart

```diff
@@ -114,6 +114,13 @@ of the iOS app that is using this module.''',
         help: 'The target platform(s) for which the app is compiled.',
         defaultsTo: Arch.values.map((arch) => arch.targetPlatformCliArg),
         allowed: Arch.values.map((arch) => arch.targetPlatformCliArg),
+      )
+      ..addOption(
+        'public-key-path',
```

Comment:
This is more of a general question but why would users need to specify the public key path to start? Wouldn't we just be using a Shorebird managed public/private key pair by default?

Reply:
We decided yesterday that we'll be asking users to manage their own keys to start.

Reply:
Okay just asked because it seems asking users to manage their own keys to start is both more work for us and more work for the users no? Either way feel free to ignore me as I haven't really been involved in the discussions around this work.

Reply:
Definitely not more work for us – it frees us from having to deal with a signing service, which is a significant chunk of work

Reply:
Got it, I was thinking we could start by having a single shorebird public/private key pair which we use for all patches for the first p.o.c which shouldn't require a signing service but if we're starting off with per-user keys then makes sense 👍